### DISCUSSION TITLE
CorsixTH 0.66

### DISCUSSION THREAD

**Discussion by TheCycoONE** [CorsixTH/CorsixTH#2358] [Team Posts]
CorsixTH 0.66 has been tagged, and a draft release page opened in GitHub. If everyone is able, we can publish the release tomorrow.

Thanks!

**Comment by rmnvgr** [comment]
The Flatpak update is ready to be merged.

**Comment by tim77** [comment]
Built for Fedora https://bodhi.fedoraproject.org/updates/?packages=corsix-th.
